.github/workflows/coverage-reports.yml

```diff
@@ -0,0 +1,118 @@
+# ------------------------------------------------------------
+# Copyright 2021 The Dapr Authors
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------
+
+name: Publishing test coverage reports of components
+
+on:
+  repository_dispatch:
+    types: [coverage-report]
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * 0'
+
+jobs:
+  coverage:
+      name: Download and merge gocov reports
+      runs-on: ubuntu-latest
+      env:
+        GOVER: "1.20"
```

Comment:
Please don't hardcode the version of Go. Instead, read it from the `go.mod` file. See example: https://github.com/dapr/components-contrib/blob/33f9c6ecef9f8f5065b6fd417e80f4a17e48a357/.github/workflows/conformance.yml#L202-L205

Reply:
Done